lib/Subscriber/Behavior/Audit/TimestampSubscriber.php

```diff
@@ -45,18 +46,23 @@ public function handleMetadataLoad($event)
             return;
         }
 
+        $encoding = 'system_localized';
+        if ($event->getMetadata()->getReflectionClass()->isSubclassOf(NonLocalizedTimestampBehavior::class)) {
+            $encoding = 'system';
+        }
```

Comment:
You could implement the logic in the CustomUrlSubscriber (assuming it exists).

I am currently doing something similar as this, but checking for the `LocaleBehavior`, which doesn't make much more sense, but would at least avoid another interface.

I think the best solution would be to make the "localized" status a property of the Metadata, so, in a subscriber for the `CustomUrlDocument`:

```
$metadata = $event->getMetadata();
$metadata->setLocalized(true);
```

or in TAML

```
Some/Bundle/Document/CustomUrlDocument
    # ...
    localized: true
```

Then you would be able to access this property from the timestampable behavior and in other places.

I can make it so.